yucca/training/trainers/YuccaConfigurator.py

```diff
@@ -18,110 +21,187 @@
 from yuccalib.network_architectures.utils.model_memory_estimation import (
     find_optimal_tensor_dims,
 )
-from yuccalib.utils.files_and_folders import WriteSegFromLogits
-from yuccalib.evaluation.loggers import TXTLogger
+from yuccalib.utils.files_and_folders import WriteSegFromLogits, load_yaml
+from yuccalib.evaluation.loggers import YuccaLogger
+from typing import Union
 
 
 class YuccaConfigurator:
     def __init__(
         self,
+        continue_from_newest_version: bool = True,
+        disable_logging: bool = False,
         folds: str = "0",
-        tiny_patch: bool = False,
         max_vram: int = 12,
         manager_name: str = None,
         model_dimensions: str = "3D",
         model_name: str = "UNet",
         planner: str = "YuccaPlanner",
         segmentation_output_dir: str = "./",
         task: str = None,
+        tiny_patch: bool = False,
     ):
+        self.continue_from_newest_version = continue_from_newest_version
         self.folds = folds
-        self.tiny_patch = tiny_patch
+        self.disable_logging = disable_logging
+        self.max_vram = max_vram
         self.model_dimensions = model_dimensions
         self.model_name = model_name
         self.manager_name = manager_name
         self.segmentation_output_dir = segmentation_output_dir
         self.planner = planner
         self.task = task
-        self.max_vram = max_vram
-        self.run_setup()
+        self.tiny_patch = tiny_patch
+
+        # Attributes set upon calling
+        self._train_split = None
+        self._val_split = None
+        self._version = None
 
-    def run_setup(self):
+        # Now run the setup
         self.setup_paths_and_plans()
-        self.setup_splits()
-        self.setup_loggers()
+        self.setup_data_params()
         self.setup_callbacks()
-        self.setup_train_params()
+        self.setup_loggers()
+
+    @property
+    def train_split(self):
+        # We do not want to make "self.load_splits" a part of the default pipeline, since many
+        # test sets will not have a training set and thus no splits. E.g. training on
+        # DatasetA with a training set and running inference on DatasetB with no training set.
+        if not self._train_split:
```

Comment:
I know `None` is falsy in Python, but I consider it a bit of a bug. I prefer
```python
if self._train_split is None:
```
since it actually checks whether `self._train_split` is what we think it is.